docs/developers/HowTo.md

```diff
@@ -125,7 +125,27 @@ To run a Scala/Java test class, you can use the below mvn command (take Velox ba
 mvn test -Pspark-3.5 -Pspark-ut -Pbackends-velox -DargLine="-Dspark.test.home=/path/to/spark/source/code/home/" -DwildcardSuites=xxx
 ```
 
-Please set `wildcardSuites` with a fully qualified class name. `spark.test.home` is optional to set. It is only required for some test suites to use Spark resources.
+Please set `wildcardSuites` with a fully qualified class name.
+
```

Comment:
I think this is unnecessary to split one line into three lines. Please keep it as it is.

Reply:
Thank you for the review, I pushed an update.